contact_map/contact_count.py

```diff
@@ -98,19 +100,51 @@ def df(self):
         columns = list(range(self.n_y))
         return pd.SparseDataFrame(mtx, index=index, columns=columns)
 
-    def plot(self, cmap='seismic', vmin=-1.0, vmax=1.0, with_colorbar=True):
+    def _check_number_of_pixels(self, figure):
+        """
+        This checks to see if the number of pixels in the figure is high enough
+        to accuratly represent the the contact map. It raises a RuntimeWarning
+        if this is not the case.
+
+        Parameters
+        ----------
+        figure: :class:`matplotlib.Figure`
+            matplotlib figure to compare the amount of pixels from
+
+        """
+        # Get dpi, and total pixelswidht and pixelheight
+        dpi = figure.get_dpi()
+        figwidth = figure.get_figwidth()
+        figheight = figure.get_figheight()
+        xpixels = dpi*figwidth
+        ypixels = dpi*figheight
+
+        # Check if every value has a pixel
+        if xpixels/self.n_x < 1 or ypixels/self.n_y < 1:
+            msg = ("The number of pixels in the figure is insufficient to show"
+                   " all the contacts.\n Please save this as a vector image "
+                   "(such as a PDF) to view the correct result.\n Another "
+                   "option is to increase the 'dpi' (currently: "+str(dpi)+"),"
+                   " or the 'figsize' (curently: "+str((figwidth, figheight)) +
+                   ").\n Adviced minimum amount of pixels = "
+                   + str((self.n_x, self.n_y))+" (width, height).")
+            warnings.warn(msg, RuntimeWarning)
+
+    def plot(self, cmap='seismic', vrange=(-1.0, 1.0), with_colorbar=True,
```

Comment:
Go ahead and keep `vmin`/`vmax`. We'll tell codeclimate that this is an exception (in general, when most arguments are minor optional tweaks of behavior, I don't worry about the too-many-arguments complaint -- so in my mind, `with_colorbar` and `kwargs` don't really count). The reason to keep them is that `vmin`/`vmax` are the standard names in matplotlib, so we want to keep closer to standard matplotlib terminology.